### ISSUE TITLE
Add if (GlobalSettings.loggingConnections)  Before LOGGING

### ISSUE THREAD

**Issue by marcobattaglia** [scalikejdbc/scalikejdbc#780]
In gclass ConnectionPool some cheks on logginnConnections are missing:
solution:

```
if (GlobalSettings.loggingConnections) {
        log.debug("Registered singleton connection pool : " + get().toString())
      }
 +    log.debug("Registered singleton connection pool : " + get().toString())
    }
```

**Comment by seratch** [comment]
I don't understand what your issue is yet.

**Comment by seratch** [comment]
Fixed by #843
